src/python/build_management/build_manager.py

```diff
@@ -487,32 +493,37 @@ def delete(self):
 class Build(BaseBuild):
   """Repesents a build type at a particular revision."""
 
-  def __init__(self, base_build_dir, revision):
+  def __init__(self, base_build_dir, revision, build_prefix='APP'):
```

Comment:
why not make build_prefix='' by default? and make prefix prepend to existing APP_* env vars?